### PR TITLE
Correctly initialize klog in clusterctl

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by vincepri** [comment]
@neolit123 This was actually the behavior before, but I removed the flag by mistake in the 1.13 refresh.

**Comment by neolit123** [comment]
ok, in that case we should cancel the hold.
/hold cancel

File: config/crds/cluster_v1alpha1_machine.yaml [hunk 1]
```diff
@@ -66,11 +66,6 @@ spec:
                           type: string
                       type: object
                   type: object
-              oneOf:
-              - required:
-                - value
-              - required:
-                - valueFrom
```
Comment (detiber) [inline config/crds/cluster_v1alpha1_machine.yaml]: This PR reverted a change previously made in #684

**Comment by neolit123** [comment]
^ @vincepri

File: config/crds/cluster_v1alpha1_machine.yaml (hunk 1 above)
Comment (vincepri) [reply config/crds/cluster_v1alpha1_machine.yaml]: I see, this is going to be replaced each time CRDs are regenereated given that kubebuilder doesn't support it.

Comment (vincepri) [reply config/crds/cluster_v1alpha1_machine.yaml]: @detiber do you know any other ways to achieve this apart from a hacky kustomize / bash script?

Comment (detiber) [reply config/crds/cluster_v1alpha1_machine.yaml]: We probably need to create and track an issue against sigs.k8s.io/controller-runtime to get a proper fix in place.

**Pull request merged**